MOxUnit/assertGreaterThan.m

```diff
@@ -0,0 +1,73 @@
+function assertGreaterThan(a, b, message)
+% assert that a is greater than b
+%
+% assertGreaterThan(a,b,[msg])
```

Comment:
Question: is it intended that a and b must be scalars? This would make sense to me, but it would be good to (1) document this explicitly and (2) ensure that the code reflects this. 

Other question: is it intended to require that a and b are numeric?

Reply:
I'll have a look on Monday to see what Matlab does in different cases so we can do something similar. It could be that they apply any or all if arrays are passed. And it would be nice to have tests succeed that also succeed in Matlab. I'll also fix the comments related to this.